examples/esp32/Cargo.toml

```
@@ -6,7 +6,7 @@ edition = "2021"
 [dependencies]
 esp-println = { version = "0.3", features = ["esp32"] }
 esp-backtrace = { version = "0.5", features = ["esp32", "panic-handler", "print-uart"] }
-hal = { package = "esp-hal", git = "https://github.com/esp-rs/esp-hal", features = ["esp32"] }
+hal = { package = "esp-hal", version = "0.19.0", features = ["esp32"] }
```

I personally prefer to omit patch numbers from dependency versions. It has the same effect of including them, but it makes it clearer that the latest `0.19.*` version is pulled.
```suggestion
hal = { package = "esp-hal", version = "0.19", features = ["esp32"] }
```